web/components/application/hacker/pages/SuccessPage/SuccessPage.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { SecondaryButton } from '../../../../input/buttons';
+
+import mountain from '../../../../../assets/circle-mountain.svg';
+import praise from '../../../../../assets/emoji/praise.svg';
+
+const SuccessPage = () => (
+  <div className="pad-nav fill-min-height flex dir-col ai-center jc-center margin-sides-l">
```

Comment:
If you change `margin-sides-l` to `pad-sides-l` the problem you're describing disappears. Other than that, looks good!

Reply:
Okay that worked thanks!!